quinn-proto/src/connection/streams/state.rs

```diff
@@ -833,25 +841,25 @@ impl StreamsState {
         let bi = id.dir() == Dir::Bi;
         // bidirectional OR (unidirectional AND NOT remote)
         if bi || !remote {
-            if remote {
-                assert!(self.send.insert(id, None).is_none());
-            } else {
-                assert!(self
-                    .send
-                    .insert(id, Some(Box::new(Send::new(self.max_send_data(&id)))))
-                    .is_none());
-            }
+            // if remote {
+            assert!(self.send.insert(id, None).is_none());
+            // } else {
+            //     assert!(self
+            //         .send
+            //         .insert(id, Some(Box::new(Send::new(self.max_send_data(&id)))))
+            //         .is_none());
+            // }
```

Comment:
Were these comments meant to be included?

Reply:
Deleted them locally but haven't pushed the commit yet, my bad!